src/Symfony/Component/DependencyInjection/Loader/XmlFileLoader.php

```diff
@@ -634,9 +634,13 @@ public function validateSchema(\DOMDocument $dom)
 EOF
         ;
 
-        $disableEntities = libxml_disable_entity_loader(false);
-        $valid = @$dom->schemaValidateSource($source);
-        libxml_disable_entity_loader($disableEntities);
+        if (LIBXML_VERSION < 20900) {
+            $disableEntities = libxml_disable_entity_loader(false);
+            $valid = @$dom->schemaValidateSource($source);
+            libxml_disable_entity_loader($disableEntities);
+        } else {
+            $valid = @$dom->schemaValidateSource($source);
```

Comment:
I'm not sure about this change: in previous code, the entity_loader were enabled, but the flag `LIBXML_NOENT` is not supported by `schemaValidateSource`.

Reply:
cf: cf https://github.com/php/php-src/pull/5867#issuecomment-664213053